app/components/Pages/HomePage/index.js

```diff
@@ -18,9 +18,15 @@ export default function HomePage({ listOfCarousels, heroMovie }) {
       <CarouselsWrapper>
         {listOfCarousels.map(({ id, name, listOfMovies }) => (
           <CarouselContainer key={id}>
-            <Link href={`/category/${id}`} passHref>
-              <CarouselHeader categoryId={id}>{name}</CarouselHeader>
-            </Link>
+            {id === 0 ? (
+              <Link href={`/mylist`} passHref>
+                <CarouselHeader categoryId={id}>{name}</CarouselHeader>
+              </Link>
+            ) : (
+              <Link href={`/category/${id}`} passHref>
+                <CarouselHeader categoryId={id}>{name}</CarouselHeader>
+              </Link>
+            )}
```

Comment:
```suggestion
            <Link href={id === 0 ? '/mylist' : `/category/${id}`} passHref>
              <CarouselHeader>{name}</CarouselHeader>
            </Link>
```
Additional notes:
1. CarouselHeader does not accept `categoryId` prop
2. CarouselHeader should extend `a`, not `p`. Otherwise, it needs to be wrapper in `<a>` or else the link doesn't work properly.